src/features/content-explorer/content-explorer/constants.js

```diff
@@ -0,0 +1,23 @@
+import { TYPE_FOLDER } from '../../../constants';
+
+/* ----------------------- Helpers for ContentExplorer ---------------------------- */
+
+export const areAllItemsActionDisabled = items => {
+    const result = items.every(item => {
+        return item.isActionDisabled;
+    });
+    return result;
+};
+
+export const isFolderPresent = items => {
+    for (let i = 0; i < items.length; i += 1) {
+        if (items[i].type === TYPE_FOLDER) {
+            return true;
+        }
+    }
+    return false;
+};
+
+export const itemsAreLoading = items => {
+    return items && items[0] && items[0].isLoading;
```

Comment:
why is it only checking the first item?

Reply:
Backend returns a hardcoded list of 7 item objects that just have the value {isLoading: true} when the items are loading. So, I just need to check that one of the items is loading. This logic was in BUIE before, I just moved it to a helper function